appium-uiautomator2-server/build.gradle

```diff
@@ -37,3 +37,17 @@ dependencies {
     androidTestCompile 'com.android.support.test.espresso:espresso-core:2.0'
 
 }
+
+task installAUT(type: Exec) {
```

Comment:
If you run `gradle connectedCheck`, then it should install the server apk & AUT.